WordPress/wordpress.org
# Theme Directory: Update packages & build process.

This updates the Gruntfile, to switch from node-sass (deprecated) to sass, removes the pixrem polyfill, and removes the es6-promise polyfill. It also updates all the dependencies to their latest versions.

See 00fd9cd87b7a71eea98ddf9b5c4ff0b8101fce39.

To test:

- Run `npm install` in the wporg-themes folder
- Run `npm run grunt build`
- The CSS files `css/style.css`, `css/style-rtl.css`, and `css/style.css.map` should be generated
- The JS file `js/theme.min.js` should be generated

There should be no functionality changes on the site.
